public/js/lib/devtools-sham/client/shared/key-shortcuts.js

```diff
@@ -141,7 +141,7 @@ KeyShortcuts.parseElectronKey = function (window, str) {
     key = ElectronKeysMapping[key];
     shortcut.keyCode = window.KeyboardEvent[key];
     // Used only to stringify the shortcut
-    shortcut.keyCodeString = key;
+    shortcut.key = key;
```

Comment:
From debugging the the keyboard shortcuts library, I noticed that the key code is mapped to DOM_VK_* key when adding to the list of events. But when the event occurs, it is not mapped properly, and so the listener is not triggered. Which is why I needed to make this change to allow it to properly map the "function" key events in the library.